python/graphstorm/gconstruct/transform.py

```diff
@@ -665,6 +666,128 @@ def call(self, feats):
 
         return {self.feat_name: feats}
 
+class NumericalStandardTransform(TwoPhaseFeatTransform):
+    """ Numerical value with standard normalization.
+
+        $val_i = \frac{val_i}{\sum_{i=0}^{N-1} val_i}$
+
+    Parameters
+    ----------
+    col_name : str
+        The name of the column that contains the feature.
+    feat_name : str
+        The feature name used in the constructed graph.
+    sum: np.array or list of floats
+        The summation of all the values in the feature col
+        from the last transformation.
+        If it is None, we will compute it on the fly.
+        Default: None
+    out_dtype:
+        The dtype of the transformed feature.
+        Default: None, we will not do data type casting.
+        Default: None
```

Comment:
Shall we specify all valid output type here?

Reply:
No. Actually, we do not check the output type.
Maybe add the check in another PR.